src/pspc/solvers/Block.tpp

```diff
@@ -185,7 +185,7 @@ namespace Pspc {
          // First, check that w[i]*ds_ is not unreasonably large:
          // (if this condition is not met, solution will have large
          // error, and user should consider using a smaller ds_)
-         UTIL_CHECK(std::abs(w[i]*ds_) < 1.0);
+         //UTIL_CHECK(std::abs(w[i]*ds_) < 1.0);
```

Comment:
Why was this commented out? This looks like a pretty benign check that you aren't doing something that would lead to horrible numerical accuracy.  Was it actually giving you trouble?

Reply:
Yeah, for whatever reason this check was getting triggered on quite a few of my test calculations, even ones that seemed to converge quite easily once this line was commented out. Others in the group have also noticed that this check fails sometimes during normal usage of PSCFPP for bulk systems, and that the calculation is able to converge if the line is commented out. I'm not entirely sure why it gives us so much grief, since it does seems like a benign check, but nevertheless it keeps coming up and I think it is probably better to take it out. We can do a more thorough investigation into why it's causing us trouble if you want.